src/constants.ts

```diff
@@ -1 +1,2 @@
 export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+export const ZERO_BYTES = ''
```

Comment:
I have a concern about this. If no content_bytes was ever sent, then yes, it returns as an empty string. However, with the RNS manager, if a user has bytes set, and then deletes them, it saves `0x00...` as the response. I can't remember 100%, but I think I was not able to set the bytes to an empty string on the contract.

[See this code here](https://github.com/rnsdomains/rns-manager-react/blob/develop/src/app/tabs/newAdmin/resolver/operations.js#L286), and then `CONTENT_BYTES_EMPTY` is defined [here](https://github.com/rnsdomains/rns-manager-react/blob/develop/src/app/tabs/newAdmin/resolver/types.js#L35).

Reply:
I guess that is the Web3.js parsing for an empty `bytes32` value. For manually parsing a `bytes` type received via RPC I just follow the specs:
1. First 32 bytes for the offset
2. Second 32 bytes for the length

I get second 32 bytes as 0 length so that is why I get `''` as empty `bytes` value